internal/kafka/test/integration/admin_kafka_test.go

```diff
@@ -1171,6 +1171,7 @@ func TestAdminKafka_Update(t *testing.T) {
 				g.Expect(resp.StatusCode).To(gomega.Equal(http.StatusOK))
 				g.Expect(result.Id).To(gomega.Equal(sampleKafkaID1))
 				g.Expect(result.Status).To(gomega.Equal(constants.KafkaRequestStatusReady.String()))
+				g.Expect(result.BootstrapServerHost).To(gomega.BeEmpty())
```

Comment:
can we ensure that the bootstrap server host is set in the database for all these kafkas https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/f9ab2b6af9c72e956df5e1833a3a2235a1b982aa/internal/kafka/test/integration/admin_kafka_test.go#L1351-L1522

in the meantime it is empty

Reply:
Thanks @machi1990, just committed the change!